problemsets-mat223/modules/module2-exercises.tex

```diff
@@ -35,7 +56,13 @@
 				coincide, or are parallel.
 			\item What is $\ell_1\cap\ell_2\cap\ell_3$?
 		\end{enumerate}
-
+		% Q3 Solutions
+		\begin{solution} 
+		    \begin{enumerate}
+		        \item $\ell_1 = \ell_3$, $\ \ell_1 \& \ell_2$ intersect, \ $\ell_3 \& \ell_2$ intersect.
```

Comment:
Don't do spaces inside of math mode. Write out "and" instead of `&`. You can use `;` to make the separation more clear if a comma is too small.